invenio_app_rdm/users_ui/templates/semantic-ui/invenio_app_rdm/users/header.html

```diff
@@ -15,7 +15,7 @@
           <img src="{{ user_avatar }}" alt="" />
         </div>
         <div class="content rel-pl-1">
-          <h4 class="header">{{ current_userprofile.full_name or current_user.username }}</h4>
+          <h4 class="header">{{ current_userprofile.full_name or current_user.username or current_userprofile.email or _('Unknown user')}}</h4>
```

Comment:
```suggestion
          <h4 class="header">{{ current_userprofile.full_name or current_user.username or current_user.email or _('Unknown user')}}</h4>
```

minor: unless I'm missing some nesting that takes place

Reply:
I think we should go with `Anonymous user` instead of Unknown, it is better for the user experience

Reply:
> I think we should go with `Anonymous user` instead of Unknown, it is better for the user experience

I agree

Reply:
> minor: unless I'm missing some nesting that takes place

did some testing, both the current option and the proposed solution work